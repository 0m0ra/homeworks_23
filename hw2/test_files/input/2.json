{
    "Lisa Roberts": {
        "region": "Moscow",
        "registered": "2015-07-05",
        "last_login": "2023-10-16",
        "email": "qcox@example.net",
        "age": 63
    },
    "Kenneth Mays": {
        "region": "Sochi",
        "registered": "2007-10-05",
        "last_login2": "2023-08-06",
        "email": "andrewswanson@example.net",
        "age": 42
    },
    "Ronnie Edwards": {
        "region": "New-York",
        "registered": "2008-09-21",
        "last_login": "2023-07-04",
        "email": "mking@example.org",
        "age": 30
    },
    "Adam Franklin": {
        "region": "New-York",
        "registered": "2005-10-09",
        "last_login": "2023-08-17",
        "email": "jonesjessica@example.org",
        "age": 65
    },
    "Dr. Angela Watkins": {
        "region": "Saint-Petersburg",
        "registered": "2001-10-17",
        "last_login": "2023-04-13",
        "email": "hamiltonwilliam@example.org",
        "age": 29
    },
    "Zachary Chase": {
        "region": "Sochi",
        "registered": "2012-10-03",
        "last_login": "2023-05-21",
        "email": "hendersonkristen@example.com",
        "age": 15
    }
}