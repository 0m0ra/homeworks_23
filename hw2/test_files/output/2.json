{"status": "Error", "details": {"msg": "Validation error for user items"}}