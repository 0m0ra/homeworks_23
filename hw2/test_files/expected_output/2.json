{"status": "Error", "details": {"msg": "Validation error for user items in data file (hw2/test_files/input/2.json): Object missing required field `last_login`"}}